internal/util/util.go

```diff
@@ -191,3 +191,40 @@ func GetClients() (*kubernetes.Clientset, *snapshotterClientSet.Clientset, error
 
 	return client, snapshotterClient, nil
 }
+
+// IsVolumeSnapshotClassHasListerSecret returns whether a volumesnapshotclass has a snapshotlister secret
```

Comment:
Minor nit: Period at the end for valid godoc string.

Reply:
added the `.` in https://github.com/vmware-tanzu/velero-plugin-for-csi/pull/37/commits/e4018c28326f338f6c346f6e8d40502d8c66ae7b